.docker/.colcon/defaults.yaml

```diff
@@ -0,0 +1,18 @@
+_common: &common
+  # Record test results to separate directory
+  # for simpler parsing and exporting with CI
+  "test-result-base": "test_results"
+
+"clean.packages":
+  <<: *common
+"build":
+  <<: *common
+  "executor": "parallel"
+  # "parallel-workers": 4
+  "symlink-install": false
+"test":
+  <<: *common
+  "executor": "parallel"
+  # "parallel-workers": 1
```

Comment:
Don't we still need this to be 1 so that texts that call ROS network interfaces don't cross pollute traffic?

Reply:
Colcon will still respect the build order DAG as defined by the package dependencies. Limiting the number of parallel-workers to one was more about avoiding system resource exhaustion when compiling. I've just relaxed this to see if we still bump into this after all the recent CMake improvements.